### PR TITLE
Tc 171 update country list

### PR DESCRIPTION
**Pull request by kevincarrogan**

### CODE REVIEW

File: assets/countries/countries-data.json [hunk 1]
```diff
@@ -1,1874 +1,1914 @@
...
-	},
-	{
-		"model": "countries.country",
-		"pk": "233",
-		"fields": {
-			"country_code": "ZW",
-			"name": "Zimbabwe"
-		}
-	}
+    {
```
Comment (andrzejlichaczewski) [inline assets/countries/countries-data.json:2]: Is the difference indentation only?

File: dit_helpdesk/countries/fixtures/countries_data.json [hunk 2]
```diff
@@ -1,1915 +1,1914 @@
...
-		"pk": "238",
-		"fields": {
-			"country_code": "XS",
-			"name": "Serbia"
-		}
-	}
-]
-
+[
+    {
```
Comment (andrzejlichaczewski) [inline dit_helpdesk/countries/fixtures/countries_data.json:2]: Same here? Also are the two countries data files equal?

File: assets/countries/countries-data.json (hunk 1 above)
Comment (kevincarrogan) [reply assets/countries/countries-data.json:2]: It's actually the type of the primary key switching from a string to an integer.

**Review by andrzejlichaczewski — APPROVED** [review]

File: dit_helpdesk/countries/fixtures/countries_data.json (hunk 2 above)
Comment (kevincarrogan) [reply dit_helpdesk/countries/fixtures/countries_data.json:2]: This is the pk changing from string to integer (same as above).

This also includes the same changes that the data migration would be doing on the live data and adding it into this fixture.

**Pull request merged**